src/main/java/roboy/linguistics/sentenceanalysis/dkpro_example/PipelineExample.java

```diff
@@ -0,0 +1,48 @@
+package roboy.linguistics.sentenceanalysis.dkpro_example;
+import de.tudarmstadt.ukp.dkpro.core.io.conll.Conll2006Writer;
+import de.tudarmstadt.ukp.dkpro.core.io.text.TextReader;
+import de.tudarmstadt.ukp.dkpro.core.languagetool.LanguageToolLemmatizer;
+import de.tudarmstadt.ukp.dkpro.core.maltparser.MaltParser;
+import de.tudarmstadt.ukp.dkpro.core.opennlp.OpenNlpPosTagger;
+import de.tudarmstadt.ukp.dkpro.core.opennlp.OpenNlpSegmenter;
+//import de.tudarmstadt.ukp.dkpro.core.clearnlp.ClearNlpSemanticRoleLabeler;
+//import de.tudarmstadt.ukp.dkpro.core.matetools.MateSemanticRoleLabeler;
+//import com.carrotsearch.hppc.ObjectIntOpenHashMap;
+import static org.apache.uima.fit.factory.AnalysisEngineFactory.createEngineDescription;
+import static org.apache.uima.fit.factory.CollectionReaderFactory.createReaderDescription;
+import static org.apache.uima.fit.pipeline.SimplePipeline.runPipeline;
+
+/**
+ * This is a linguistic analysis pipeline example from: https://dkpro.github.io/dkpro-core/pages/java-intro/
+ *
+ * If you wish to test the pipeline:
+ * 1) uncomment the DKPro dependencies in pom.xml
+ * 2) write your input into the file document.txt
+ * 3) The output will be written in document.txt.conll
+ *
+ * If you wish to continue integrating SRL tools, you should start by
+ * uncommenting the additional dependencies in pom and under imports.
+ * At the time of writing, neither of the labelers worked:
+ * ClearNlp SRL does not produce output, and MateSRL throws a
+ * ClassNotFoundException - possibly due to some dependency issues.
+ *
+ * @author Laura Lahesoo
+ */
+
+public class PipelineExample {
+
+    public static void main(String[] args) throws Exception {
+        runPipeline(
+                createReaderDescription(TextReader.class,
+                        TextReader.PARAM_SOURCE_LOCATION, "src/main/java/roboy/linguistics/sentenceanalysis/dkpro_example/document.txt",
```

Comment:
don't read from the file, but add text to parse as input argument to the function